client/client.go

```diff
@@ -27,6 +29,15 @@ var (
 		MaxIdleTimeout:  20 * time.Second,
 		KeepAlivePeriod: 10 * time.Second,
 	}
+
+	// DefaultBackoff is the default backoff used when dialing and serving
+	// a connection.
+	DefaultBackoff = wait.Backoff{
```

Comment:
would we want to make this configurable in the future?

Reply:
Yeah probably, ill leave it off for now, but but eventually parameters on the server probably makes sense.